src/App.vue

```diff
@@ -541,8 +542,8 @@ export default {
       const email: string = userInfo.contacts[0]?.email || userInfo.email
       const firstName: string = userInfo?.firstname
       const lastName: string = userInfo?.lastname
-      // remove leading { and trailing } and tokenize string
-      const custom: any = { roles: userInfo.roles?.slice(1, -1).split(',') }
```

Comment:
Hey Severin, would it be better to update the backend call to return the roles here? That way you wouldn't need to make this change across the other UIs and anything else using the same flow would automatically get this change in their ldarkly users

Reply:
Or maybe you can remove calling the api for the userInfo altogether? I think all of the information is within the JWT which you are already parsing within the UI. Maybe there's another reason I'm not aware of

Reply:
The UI gets the feature flags in the `user` response from LD.

Eg, payload:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/a52666f9-33e2-49a2-925f-15075a79e5c6)

Eg, response:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/11fbc3b5-4e8e-41c1-9e35-db4bcd0ba196)

I'm afraid both the UI and the API will need to "identify" the user. I don't know whether the user's attributes is retained between `user` calls. For example, if the API provides a custom object but the UI doesn't, will LD use the custom fields from the first call?

Reply:
Yes, it looks like the UI can obtain most, if not all, the user info from the JWT, so fetching the user info from the Auth API (`@me`) may not be needed.

Reply:
LMK if OK to merge or if you want some changes.